main.css

```diff
@@ -0,0 +1,209 @@
+/*
+===========================
+Body styles
+===========================
+*/
+
+.master {
+  background: linear-gradient(to bottom, #3cd6ff 0%, #ff4fe3 100%);
+}
+
+header {
+  margin-top: 10px;
+  display: flex;
+  justify-content: center;
+  font-size: 50px;
+  font-family: Arial;
+  font-weight: 900;
+  color: #ffffff;
+  text-shadow: 3px 3px 8px #2a2b2a;
+}
+
+/*
+===========================
+Grid styles
+===========================
+*/
+
+.canvas {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 200px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  height: 600px;
+  width: 1000px;
+  border-style: solid;
+  border-width: 8px;
+  border-color: #695cd6;
+}
+
+.grid {
+  box-sizing: border-box;
+  height: 20px;
+  width: 20px;
+  background-color: #ffffff;
+  border-style: solid;
+  border-width: 1px;
+  border-color: #d4d4d4;
+}
+
+
+
+
+/*
+===========================
+Palette styles
+===========================
+*/
+
+.palette {
+  margin-bottom: 30px;
+}
+
+.blackButton {
+  margin-left: 300px;
+  margin-right: 10px;
+  border-width: 3px;
+  border-radius: 5px;
+  border-color: #ffffff;
+  height: 30px;
+  width: 60px;
+  background-color: #000000;
+}
+
+.grayButton {
+  margin-left: 10px;
+  margin-right: 10px;
+  border-width: 3px;
+  border-radius: 5px;
+  border-color: #ffffff;
+  height: 30px;
+  width: 60px;
+  background-color: #6f6f6f;
+}
+
+.whiteButton {
+  margin-left: 10px;
+  margin-right: 10px;
+  border-width: 3px;
+  border-radius: 5px;
+  border-color: #ffffff;
+  height: 30px;
+  width: 60px;
+  background-color: #ffffff;
+}
+
+.redButton {
+  margin-left: 10px;
+  margin-right: 10px;
+  border-width: 3px;
+  border-radius: 5px;
+  border-color: #ffffff;
+  height: 30px;
+  width: 60px;
+  background-color: #fe3f3f;
+}
+
+.blueButton {
+  margin-left: 10px;
+  margin-right: 10px;
+  border-width: 3px;
+  border-radius: 5px;
+  border-color: #ffffff;
+  height: 30px;
+  width: 60px;
+  background-color: #299fff;
+}
+
+.greenButton {
+  margin-left: 10px;
+  margin-right: 10px;
+  border-width: 3px;
```

Comment:
seeing the same styles a lot across all these buttons. You could DRY this up by creating a .btn class that has all the "standard" button styles, and then apply the unique styles in a different class.